components/suggest/src/schema.rs

```diff
@@ -96,7 +110,7 @@ impl ConnectionInitializer for SuggestConnectionInitializer {
 
     fn upgrade_from(&self, _db: &Transaction<'_>, version: u32) -> open_database::Result<()> {
         match version {
-            1..=4 => {
+            1..=6 => {
```

Comment:
```suggestion
            1..=5 => {
```

Let's make this one version lower than the current. `upgrade_from` shouldn't be called if the database version  is current, but we want that to fail nosily with an error if it does.